composer.json

```diff
@@ -12,39 +12,86 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/installers": true,
+            "drupal/core-composer-scaffold": true,
+            "cweagans/composer-patches": true,
+            "zaporylie/composer-drupal-optimizations": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true,
+            "oomphinc/composer-installers-extender": true,
+            "php-http/discovery": true,
+            "drupal/core-project-message": true,
+            "phpstan/extension-installer": true
+        }
     },
-    "repositories": {
-        "drupal-8": {
+    "repositories": [
+        {
+            "type": "package",
+            "package": {
+                "name": "drupal/google_cloud_vision",
+                "type": "drupal-module",
+                "version": "dev-8.x-1.x",
+                "source": {
+                    "type": "git",
+                    "url": "https://git.drupalcode.org/issue/google_cloud_vision-3427974.git",
+                    "reference": "92827a71e881cf9ab4e5de7e13cf5c007bacd75e"
+                }
+            }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "drupal/media_entity_video",
+                "type": "drupal-module",
+                "version": "dev-8.x-1.x",
+                "source": {
+                    "type": "git",
+                    "url": "https://git.drupalcode.org/issue/media_entity_video-3297609.git",
+                    "reference": "c9aec85065b038709a4977edda337a65efd3de7c"
+                }
+            }
+        },
+        {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         },
-        "asset-packagist": {
+        {
             "type": "composer",
             "url": "https://asset-packagist.org"
         }
-    },
+    ],
     "require": {
-        "php": "^7.2",
-        "brainsum/damo-distribution": ">=0.26 <1.0",
+        "php": ">=7.4",
```

Comment:
https://github.com/drupal-composer/drupal-project/blob/10.x/composer.json
alapjan itt mar 8as phpt kellene hivni.
Lehet erdemes megnezni a tobbi dolgot is abban a composer.json-ban es frissiteni